### PR TITLE
Optional Favicon Injection

### PR DESCRIPTION
**Pull request by TurtIeSocks**
- Inspired by favicon injection offered by the Webpack HTML plugin:
https://github.com/jantimon/html-webpack-plugin

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -192,6 +193,16 @@ export const htmlPlugin = (configuration: Configuration = { files: [], }): esbui
                         document.title = htmlFileConfiguration.title
                     }
 
+                    if (htmlFileConfiguration.favicon) {
+                        // Injects a favicon if present
+                        fs.copyFile( htmlFileConfiguration.favicon, `${outdir}/favicon.ico` )
```
Comment (craftamap) [inline src/index.ts:198]: copyFile is async, so it needs to be awaited here. :)

```suggestion
                        await fs.copyFile( htmlFileConfiguration.favicon, `${outdir}/favicon.ico` )
```

**Review by craftamap — CHANGES_REQUESTED** [review]
Thanks for the contribution! I find one small error, otherwise it's looking really good!

**Comment by TurtIeSocks** [comment]
All set in latest, good catch 😄

**Comment by craftamap** [comment]
Looks like the sources weren't rebuilt after the recent changes, so I pushed them onto your branch. Looking good now!

**Review by craftamap — APPROVED** [review]

**Comment by craftamap** [comment]
Thanks for the contribution!

**Pull request merged**